src/main/java/com/strandls/esmodule/services/impl/ElasticSearchServiceImpl.java

```diff
@@ -562,6 +584,51 @@ public AggregationResponse aggregation(String index, String type, MapSearchQuery
 		return aggregationResponse;
 	}
 
+	public List<UploadersInfo> uploaderInfo(String index, String userIds) {
+
+		List<String> l = Arrays.asList(userIds.split(","));
+
+		List<UploadersInfo> result = new ArrayList<>();
+
+		for (int i = 0; i < l.size(); i++) {
+			String id = l.get(i);
+
+			BoolQueryBuilder boolQueryBuilder = QueryBuilders.boolQuery()
+					.must(QueryBuilders.termQuery("author_id", id));
+
+			SearchSourceBuilder sourceBuilder = new SearchSourceBuilder();
+
+			sourceBuilder.query(boolQueryBuilder);
+			sourceBuilder.size(1);
+
+			SearchRequest request = new SearchRequest(index);
+			request.source(sourceBuilder);
+			SearchResponse response;
+			try {
+				response = client.search(request, RequestOptions.DEFAULT);
+				for (SearchHit hit : response.getHits().getHits()) {
+
+					Map<String, Object> sourceMap = hit.getSourceAsMap();
+
+					String name = String.valueOf(sourceMap.get("created_by"));
+					String pic = String.valueOf(sourceMap.get("profile_pic"));
+
+					Long authorId = Long.parseLong(String.valueOf(sourceMap.get("author_id")));
+
+					UploadersInfo uploaderInfo = new UploadersInfo(name, pic, authorId);
+
+					result.add(uploaderInfo);
+
+				}
+			} catch (IOException e) { // TODO Auto-generated catch block
+				e.printStackTrace();
```

Comment:
remove printstackstrace with logger, and use exception catch at the last

Reply:
added logger